lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/InitialiseIpvSessionHandler.java

```diff
@@ -175,7 +184,19 @@ public APIGatewayProxyResponseEvent handleRequest(
             String clientOAuthSessionId = SecureTokenHelper.getInstance().generate();
 
             IpvSessionItem ipvSessionItem =
-                    ipvSessionService.generateIpvSession(clientOAuthSessionId, null, emailAddress);
+                    ipvSessionService.generateIpvSession(
+                            clientOAuthSessionId, null, emailAddress, isReverification);
```

Comment:
I'm not sure if it's there's a workaround to get onto a prohibited journey but could make more sense to put validation before this, and use the isReverification for the validation instead

Reply:
I've refactored to validate the scope rather than the journey - this way we can validate it before we get here :)